features/features/package_managers/bundler_spec.rb

```diff
@@ -4,11 +4,17 @@
 require_relative '../../../lib/license_finder/version'
 
 describe 'Bundler Dependencies' do
+  around do |example|
+    # Run license_finder outside any Bundler environment
+    ::Bundler.with_unbundled_env do
```

Comment:
Bundler leaks environment variables, so these tests need to be isolated. Bundler environment variables end up affecting license_finder executed even from the shell command (talking about `BUNDLER_` not `BUNDLE_` variables)